app/javascript/src/components/Dashboard/Contacts/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+
+import { Button, Pagination, Table } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import { TABLE_ROW_DATA } from "./constants";
+import Menu from "./Menu";
+import { Column } from "./Table/Column";
+
+const Contacts = () => {
+  const [searchTerm, setSearchTerm] = useState("");
+  const [openMenuBar, setOpenMenuBar] = useState(true);
```

Comment:
Why do we need to open the menu bar when we are visiting or rendering the `Contacts` components for the first time?

Also name this as `showMenuBar`.

Reply:
According to Figma design the menubar was opened. That's why I set the menu as open. I shall change this.

Reply:
Thanks for pointing that out. Will update the FIgma.